### PR TITLE
Added scripts for the 10-28 hoc-engineers mailings

### PR DESCRIPTION
**Pull request by aoby**
Output of generate-hoc-engineers on my local machine:

```
0 unsubscribers loaded.
query:
kind_s:VolunteerEngineerSubmission2015
3365 hoc-engineers-volunteers
query:
kind_s:HelpUs2013 || kind_s:VolunteerEngineerSubmission || (kind_s: Petition && role_s: "engineer")
119666 hoc-engineers (initial query)
120186 including mailchimp engineers
```

### CODE REVIEW

File: bin/mailing-common/mailing-list-utils.rb [hunk 1]
```diff
@@ -74,3 +74,27 @@ def query_subscribed_contacts(params)
     ALL.merge! results
   end
 end
+
+EMAIL = 'Email Address'
+NAME = 'Name'
+COUNTRY_CODE = 'CC'
+COUNTRY_CODE_US = 'US'
+
+def include_mailchimp_engineers(results)
```
Comment (laurelfan) [inline bin/mailing-common/mailing-list-utils.rb:83]: if this method used ALL then I think you wouldn't need to reject! the volunteers again.

Comment (aoby) [inline bin/mailing-common/mailing-list-utils.rb:83]: good point. updated

Comment (laurelfan) [inline bin/mailing-common/mailing-list-utils.rb:83]: ALL is kind of weird though.. but this is going away soon when you rewrite the mail system, right? :)

are the numbers the same as before?

**Comment by laurelfan** [comment]
LGTM

File: bin/mailing-common/mailing-list-utils.rb (hunk 1 above)
Comment (aoby) [inline bin/mailing-common/mailing-list-utils.rb:83]: Agreed. ALL was leftover from the old scripts. I plan on improving / removing most of this, and coming up with a place to store the generated csv files :)

Yes, the numbers are the same.
- Before:

```
119666 hoc-engineers (initial query)
120655 including mailchimp engineers
120186 hoc-engineers (excluding volunteers)
```
- After:

```
119666 hoc-engineers (initial query, minus hoc-engineers-volunteers)
120186 including mailchimp engineers
```

**Pull request merged**